src/dist/docbook-xsl/fo-pdf.xsl

```diff
@@ -48,11 +48,11 @@
   -->
 
   <xsl:template name="pickfont-sans">
-    <xsl:text>Arial,sans-serif</xsl:text>
+    <xsl:text>Arial,Liberation Sans,sans-serif</xsl:text>
   </xsl:template>
 
   <xsl:template name="pickfont-serif">
-    <xsl:text>Georgia,serif</xsl:text>
+    <xsl:text>Georgia,DejaVu Serif,Liberation Serif,serif</xsl:text>
```

Comment:
We should either add DejaVu Sans to pickfont-sans or remove DejaVu Serif from pickfont-serif. We shouldn't have one set of fallbacks for one family and a different one for another family.